packages/widgets/src/menubar.ts

```diff
@@ -947,6 +973,15 @@ export namespace MenuBar {
      */
     readonly tabbable: boolean;
 
+    /**
+     * Whether the item is disabled.
+     *
+     * #### Notes
+     * A disabled item cannot be active.
+     * A disabled item cannot be focussed.
+     */
+    readonly disabled: boolean;
```

Comment:
As this is new, we must make it optional to keep the API backward compatible:

```suggestion
    readonly disabled?: boolean;
```

Reply:
good catch, thank you!